src/data.json

```diff
@@ -165,7 +165,8 @@
     ],
     "linkedin": "linkedin.com/in/redacted",
     "github": "github.com/redacted",
-    "twitter": "x.com/redacted"
+    "twitter": "x.com/redacted",
+    "website": "website/portfolio"
```

Comment:
What do you think about refactoring this part into:
```json
{ 
  "links" : [
    { "type": "LinkedIn", "link": "..." } , 
    { "type": "LinkedIn", "link": "..." } 
  ]
}
  ```
This will allow to use a map in `Profile.tsx` and to narrow the supported icons via the type in `ProfileLink.tsx`.

Reply:
Good point! I’ll make those changes.